.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+
+You can update, or add content to, existing topics in their respective versioned directories (for 2.0, 2.1, 2.2, and onward).
+
+The following guidelines may answer most of your questions and help you get started:
+
+1.  Check [existing pull requests](https://GitHub.com/magento/devdocs/pulls){:target=\_blank"} and make sure you are not duplicating work!
+
+1.  For large contributions or changes that include multiple files, [open an issue](#issues) and discuss it with us first. This may further prevent duplicate or unnecessary effort.
+
+1.  Familiarize yourself with the existing documentation. Look through and search the guides to decide where to add your topics.
+
+    -   The DevDocs team can find the best home for your new topics and add it to the navigation.
+    -   If a topic has a symlink, you can remove it with git commands and add a new file. Copy and paste a previous version of the topic to get started.
+
+1.  Focus on the content and on creating useful information for your fellow Magento developers and community members. Don't forget to review your work for typos, formatting errors, or sentences that need clarifying before opening a pull request.
+
+1.  Use the following guidelines to help you with the writing process:
+
+    -   Define the goal of your topic. What exactly do you want to teach the reader?
+    -   Make the title of your topic reflect the content.
+    -   Keep your sentences concise and try to separate conceptual information from procedural steps.
+    -   Remember to use active voice (not passive), write in the present tense, and use a friendly tone in second person. For example, _"You can now view the output..."_.
+    -   Use notes to alert readers about important details.
+    -   Use cross-references to other topics if appropriate. We can help you with the syntax if it is not clear. The template provides an example you can use.
+
+## Fork and clone a repository {#fork}
+
+To fork the devdocs repository on GitHub, do the following:
+
+1.  Create or log into your development environment account on GitHub.
+
+1.  Navigate to the <a href="https://GitHub.com/magento/devdocs" target="\_blank">DevDocs repository</a>.
+
+1.  Click **Fork** at the top right.
+
+1.  Clone the repository into your development environment and start writing. Commit changes and  and committing your changes. Optionally, create a branch first if you plan to work on multiple changes.
+
+1.  Before opening a pull request, don't forget to review your work for typos, formatting errors, code sample formatting, or sentences that need clarifying.
+
+### Preview HTML locally
+
+To preview your changes in HTML output, follow the instructions in the [README](https://GitHub.com/magento/devdocs/blob/develop/README.md){:target="_blank"} to build the devdocs site locally using Jekyll.
+
+### Update your fork {#sync}
```

Comment:
Let's link to GitHub's docs on [syncing a fork](https://help.github.com/articles/syncing-a-fork/) instead of providing instructions. It'll help simplify this topic.

Reply:
Will do, getting this added in next commit